src/components/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './index.css'
+
+
+const btnStyle = {
+  backgroundColor: '#FFEBD9',
+  color: "black"
+};
+
+function Button(props) {
+  return (
+    <div>
+      <button
+        className="btn"
+        style={(props.mark && props.symbol === props.mark.toLowerCase())? btnStyle : []}
```

Comment:
You can ensure that there is always a default prop value by using the [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) library.